web-client/iron-remote-desktop/src/iron-remote-desktop.svelte

```diff
@@ -16,12 +16,16 @@
 <script lang="ts">
     import { onMount } from 'svelte';
     import { loggingService } from './services/logging.service';
-    import { WasmBridgeService } from './services/wasm-bridge.service';
+    import { type RemoteDesktopModule, WasmBridgeService } from './services/wasm-bridge.service';
     import { LogType } from './enums/LogType';
     import type { ResizeEvent } from './interfaces/ResizeEvent';
     import { PublicAPI } from './services/PublicAPI';
     import { ScreenScale } from './enums/ScreenScale';
-    import { ClipboardContent, ClipboardTransaction } from '../../../crates/ironrdp-web/pkg/ironrdp_web';
+    import type { ClipboardTransaction as IClipboardTransaction } from './interfaces/ClipboardTransaction';
+    import { ClipboardContent, ClipboardTransaction } from './../../iron-remote-desktop-rdp/src/main';
+    import * as remote_desktop from './../../iron-remote-desktop-rdp/src/main';
+    // import { ClipboardContent, ClipboardTransaction } from './../../../../ironvnc/web-client/iron-remote-desktop-vnc/src/main';
```

Comment:
Here as you can see, we can import VNC instead of an RDP and this will inject selected module into a WasmBridgeService

Reply:
Wonderful!

Reply:
Wait, I was expecting this logic to be in iron-svelte-client, not iron-remote-desktop.
Is it possible to choose between injecting IronRDP or injecting IronVNC from iron-svelte-client?

Reply:
Change the logic to do dependency injection in  iron-svelte-client